### ISSUE TITLE
flux positivity constraint

### ISSUE THREAD

**Issue by benjaminpope** [benjaminpope/drpangloss#23]
It is a well-known phenomenon that for a real binary signal, the best fitting binary has positive flux on one side and negative on another.

Here is AB Dor: 

![image](https://github.com/benjaminpope/drpangloss/assets/6024213/4edb6976-3b37-4779-8484-81af7dff65c4)

It looks bad because after optimization it is negative on one side and positive on the other. I think we should do this calculation in log flux so that it is strictly positive, but I wanted to check @shashankdholakia and @LouisDesdoigts agree.

**Comment by LouisDesdoigts** [comment]
Yeah I can't see why not (beyond maybe the negative flux plot looking cooler)
